WalletWasabi.Fluent/Helpers/TransactionHelpers.cs

```diff
@@ -173,7 +173,8 @@ public static async Task<SmartTransaction> ParseTransactionAsync(string path, Ne
 	public static async Task<bool> ExportTransactionToBinaryAsync(BuildTransactionResult transaction)
 	{
 		var psbtExtension = "psbt";
-		var filePath = await FileDialogHelper.ShowSaveFileDialogAsync("Export transaction", new[] { psbtExtension });
+		string initialFileName = transaction.Transaction.GetHash().ToString()[..5];
```

Comment:
I would increase that to 7 for example like the short version of commit ids.
https://stackoverflow.com/questions/18134627/how-much-of-a-git-sha-is-generally-considered-necessary-to-uniquely-identify-a

```suggestion
		string initialFileName = transaction.Transaction.GetHash().ToString()[..7];
```

Reply:
> What is the argument against storing complete txid?

Nothing actually.

> I mean user is free to make it shorter as he/she wants

So true.

I will make this change soon.